client/blocks/reader-post-card/compact.jsx

```diff
@@ -22,10 +24,16 @@ const CompactPost = ( {
 			: null;
 
 	const isSmallScreen = useBreakpoint( '<660px' );
+	const [ hasExcerpt, setHasExcerpt ] = useState( false );
```

Comment:
Should we default `hasExcerpt` to be `true` so we can avoid jiggles/shifts from updating it later?
```suggestion
	const [ hasExcerpt, setHasExcerpt ] = useState( true );
```